### DISCUSSION TITLE
Away mode doesn't allow changes

### DISCUSSION THREAD

**Discussion by loucks1** [PeteRager/lennoxs30#344] [Q&A]
I'm using 2024.8.0 - when I attempt to change the set temp or fan mode when in away mode, nothing happens (technically, HA sees the change and shows that it was accepted, but it reverts to whatever is set at the thermostat within a minute (seems to be the result of a poll of the thermostat). 

If I attempt change it via the iComfort app, it takes the change and syncs to HA.  

Is this a known issue or working as designed?  I didn't see any previous discussions around this.

**Comment by PeteRager** [comment]
This sounds like a bug,  What would be helpful is to turn on message logging, restart HA, put the system into away mode.  Then

a) try the setpoint from HA - record the number and approximate time you did this - so I can correlate with message log
b) a minute later, do the same from iComfort App

repeat with fan mode, post the log.

**Reply by PeteRager** [reply]
And please create an issue so we can track this.
